src/backend/contacts.nim

```diff
@@ -13,7 +13,7 @@ proc getContactById*(id: string): RpcResponse[JsonNode] {.raises: [Exception].}
   result = callPrivateRPC("getContactByID".prefix, payload)
 
 proc blockContact*(id: string): RpcResponse[JsonNode] {.raises: [Exception].} =
-  result = callPrivateRPC("blockContact".prefix, %* [id])
+  result = callPrivateRPC("blockContactDesktop".prefix, %* [id])
```

Comment:
I have a feeling that this name is not good since it is not descriptive at all. If we need some different functionality than basic blockContact, it should be named "blockContactAndDoSth...". Adding a new versions of functions with adding ...Desktop is not a good way in my opinion. The same refers to changes in go layer.

Reply:
@endulab `blockContactDesktop` name is here just to signify that it should be used instead `blockContact` call for the desktop app. Basically it's the same as `blockContact` but just doesn't set `Added` flag and doesn't delete chat for the blocked user from the db. 

Furthermore... this is temporary here till we sort out contacts related flags on the `status-go` side and after that we will remove it.